docs/ExpoManagedWorkflow.md

```diff
@@ -29,6 +29,7 @@ const AppcuesWrapper = NativeModules.AppcuesReactNative ?? {
   show: (experienceID) => { console.log(`Appcues.show(${experienceID})`) },
   debug: () => { console.log(`Appcues.debug()`) },
   didHandleURL: (url) => { console.log(`Appcues.didHandleURL(${url})`); return false },
+  AppcuesFrameView: View,
```

Comment:
@mmaatttt I'd rather get this in here so I don't forget. Is this how you would do it? fallback to a `View`? This is the first non function export in here.

Reply:
I don't think it's this simple since `AppcuesFrameView` isn't in `NativeModules.AppcuesReactNative`. Not sure what the right approach is, but I'll figure it out

Reply:
Pushed a new commit to this PR that should work. Do you mind giving it a try to make sure it's clear?

Reply:
nice! confused a little on this part
```
export const WrappedAppcuesFrameView =
  UIManager.getViewManagerConfig('AppcuesFrameView') == null
    ? require('@appcues/react-native').AppcuesFrameView
    : PlaceholderFrameView;
```

should that condition be `!= null` ? (or the true/false cases flipped?)